src/main/java/com/chat/yourway/controller/ChatController.java

```diff
@@ -11,29 +11,23 @@
 
 import java.security.Principal;
 
-/**
- * {@link ChatController}
- *
- * @author Dmytro Trotsenko on 7/21/23
- */
-
 @Controller
 @RequiredArgsConstructor
 public class ChatController {
 
-    private final MessageService messageService;
+  private final ChatMessageServiceImpl chatMessageServiceImpl;
```

Comment:
Could you use interface here instead of implementation? I mean everywhere in code at the left side when we declare variable we should avoid using implementation type